src/Symfony/Bridge/PhpAmqp/Connection.php

```diff
@@ -0,0 +1,192 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\PhpAmqp;
+
+/**
+ * An AMQP connection.
+ *
+ * @author Samuel Roze <samuel.roze@gmail.com>
+ */
+class Connection
+{
+    private $amqpConnectionCredentials;
```

Comment:
Rename this to `$amqpCredentials` to be consistent with the other vars? `$amqpChannel`, `$amqpExchange`, etc.

Reply:
Or maybe `$connectionCredentials` ? All the variables prefixed by `amqp` (except this one) actually contains objects coming from PHP's AMQP library.